### PR TITLE
revert varnish images to original repo for correct versioning

### PR DESCRIPTION
**Pull request by erlisdhima**

### CODE REVIEW

File: .github/workflows/docker-image-varnish.yml [hunk 1]
```diff
@@ -1,213 +1,52 @@
...
+
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - "6.0"
+          - "6.5"
+          - "6.6"
+          - "7.0"
+          - "7.1"
```
Comment (drpayyne) [inline .github/workflows/docker-image-varnish.yml:23]: Should the versions be `xy` instead of `x.y`?

File: images/varnish/Dockerfile [hunk 2]
```diff
@@ -4,9 +4,11 @@ RUN dnf upgrade -y \
     && dnf clean all \
     && rm -rf /var/cache/dnf
 
-ARG RPM_SCRIPT=https://packagecloud.io/install/repositories/varnishcache/varnish41/script.rpm.sh
+ARG VARNISH_VERSION=60lts
+ARG RPM_SCRIPT=https://packagecloud.io/install/repositories/varnishcache/varnish${VARNISH_VERSION}/script.rpm.sh
 RUN curl -s "${RPM_SCRIPT}" | bash \
     && dnf install -y epel-release \
+    && dnf module disable -y varnish \
```
Comment (drpayyne) [inline images/varnish/Dockerfile:11]: What's the reason for disabling this module?

**Review by drpayyne — CHANGES_REQUESTED** [review]

File: .github/workflows/docker-image-varnish.yml (hunk 1 above)
Comment (erlisdhima) [reply .github/workflows/docker-image-varnish.yml:23]: The `.` is removed in this line https://github.com/drpayyne/warden-multi-arch/pull/32/files#diff-876f88dab6320ed343f8b53555ad6366f7c787f34b0daa66cba4ced9fd5216fcR41

File: images/varnish/Dockerfile (hunk 2 above)
Comment (erlisdhima) [reply images/varnish/Dockerfile:11]: image
More info here: https://varnish-cache.org/docs/trunk/installation/install_redhat.html

Comment (drpayyne) [reply images/varnish/Dockerfile:11]: Thank you!

**Comment by drpayyne** [comment]
Merging now!

**Pull request merged**